data/projects.js

```diff
@@ -361,5 +361,20 @@ coala should support generating metrics for your code.",
 		"tags" : ["Bears", "cib"],
 		"markdown": "improve_installation.md",
 		"collaborating_projects" : ["coala"]
-	}
+	},
+	{
+		"name" : "Create coala-GUI",
+		"desc" : "Crate a GUI for coala so that the user can analyse the \
+		 code in the most easiest way without worrying about any more commands.",
+		"requirements" : [
+				"The proposal needs to contain a clear concept"
+		],
+		"difficulty" : "medium/high",
+		"issues" : [],
+		"mentors" : ["@Udayan12167", "@sils"],
```

Comment:
I'm not mentoring this and I'm voting against this project happening at all. coala is already fragmented a lot and coala-html serves a similar purpose, we should focus on that one indeed. Not trying to be mean but sorry, I'm totally against this.